webrtc.html

```diff
@@ -853,6 +853,12 @@
                   </li>
 
                   <li>
+                    <p>If the supplied description matches the state of all
+                    tracks, clear the negotiation-needed flag.
```

Comment:
You got it.

Changing the RtpSender might work.  Perhaps the best thing to do is to open an issue and address it separately.